src/rebar_app_info.erl

```diff
@@ -72,7 +72,7 @@
                      app_file_src       :: file:filename_all() | undefined,
                      app_file_src_script:: file:filename_all() | undefined,
                      app_file           :: file:filename_all() | undefined,
-                     original_vsn       :: binary() | string() | undefined,
+                     original_vsn       :: binary() | undefined,
```

Comment:
I'll look around.

Really want to shrink types like this. We have a lot of places where we have to deal with different types because of the different between user input and internal representation or hex registry types.